DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/register_map_and_streaming_interfaces/src/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# To see a Makefile equivalent of this build system:
+# https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC++/ProjectTemplates/makefile-fpga
+
+set(SOURCE_FILE register_map_and_streaming_interfaces.cpp)
+set(TARGET_NAME register_map_and_streaming_interfaces)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
+
+# FPGA device selection
+if(NOT DEFINED FPGA_DEVICE)
+    set(FPGA_DEVICE "Arria10")
+    message(STATUS "FPGA_DEVICE was not specified.\
+                    \nConfiguring the design to run on the default FPGA device ${FPGA_DEVICE} (Intel(R) Arria(R) 10 FPGAs). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA device ${FPGA_DEVICE}")
+endif()
+
+# This is a Windows-specific flag that enables exception handling in host code
+if(WIN32)
+    set(WIN_FLAG "/EHsc")
+endif()
+
+# This is for finding the install path to the necessary include files
+if(DEFINED ENV{INTELFPGAOCLSDKROOT})
```

Comment:
This should not be necessary, why do you need this?

Reply:
This is needed because there's an extra command line flag needed to point to the interface control header file
icpx -I/$INTELFPGAOCLSDKROOT/include